ibc-testkit/src/fixtures/clients/tendermint.rs

```diff
@@ -172,9 +174,28 @@ pub fn dummy_ics07_header() -> Header {
     }
 }
 
+#[cfg(feature = "serde")]
+pub fn test_serialization_roundtrip<T>(json_data: &str)
+where
+    T: core::fmt::Debug + PartialEq + Serialize + DeserializeOwned,
+{
+    let parsed0 = serde_json::from_str::<T>(json_data);
+    assert!(parsed0.is_ok());
+    let parsed0 = parsed0.expect("should not fail");
+
+    let serialized = serde_json::to_string(&parsed0);
+    assert!(serialized.is_ok());
+    let serialized = serialized.expect("should not fail");
```

Comment:
can we add an assert for the serialized string too?

```rs
assert_eq!(serialized, json_data);
```

Reply:
[a9dc193](https://github.com/cosmos/ibc-rs/pull/1233/commits/a9dc193f7e3a4df0b6242291ca3b8751e1768f2f)